espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/GetDisplayed.kt

```diff
@@ -14,24 +14,25 @@
  * limitations under the License.
  */
 
-package io.appium.espressoserver.lib.handlers;
+package io.appium.espressoserver.lib.handlers
 
-import io.appium.espressoserver.lib.handlers.exceptions.AppiumException;
-import io.appium.espressoserver.lib.handlers.exceptions.StaleElementException;
-import io.appium.espressoserver.lib.model.AppiumParams;
-import io.appium.espressoserver.lib.model.Element;
+import io.appium.espressoserver.lib.handlers.exceptions.AppiumException
+import io.appium.espressoserver.lib.handlers.exceptions.StaleElementException
+import io.appium.espressoserver.lib.model.AppiumParams
+import io.appium.espressoserver.lib.model.Element
 
-public class GetDisplayed implements RequestHandler<AppiumParams, Boolean> {
+class GetDisplayed : RequestHandler<AppiumParams, Boolean> {
 
-    @Override
-    public Boolean handle(AppiumParams params) throws AppiumException {
+    @Throws(AppiumException::class)
+    override fun handle(params: AppiumParams): Boolean? {
```

Comment:
Boolean? -> Boolean

Reply:
I have another PR incoming that fixes all these.